src/luarocks/build.lua

```diff
@@ -436,7 +438,8 @@ function build.build_rockspec(rockspec, opts)
    ok, err = repos.run_hook(rockspec, "post_install")
    if not ok then return nil, err end
 
-   util.announce_install(rockspec)
+   local cmd = require("luarocks.cmd")
+   cmd.announce_install(rockspec)
```

Comment:
Would it be possible to move some of these messages out of these internal module functions and into the `cmd` functions that call them?

No need to handle them all yet, but at least some of these messages that run at the very beginning or the very end of these internal functions should be easy to move out into the cmd caller.

Reply:
Yeah, sure, I'll do it!